.github/workflows/build.yml

```diff
@@ -83,6 +83,13 @@ jobs:
         docker tag antrea/antrea-agent-ubuntu:"${DOCKER_TAG}" antrea/antrea-agent-ubuntu-${{ matrix.suffix }}:"${DOCKER_TAG}"
         docker push antrea/antrea-controller-ubuntu-${{ matrix.suffix }}:"${DOCKER_TAG}"
         docker push antrea/antrea-agent-ubuntu-${{ matrix.suffix }}:"${DOCKER_TAG}"
+    # Same repository workflow dispatch to run conformance tests
```

Comment:
will it run conformance tests 3 times for every commit?

Reply:
Yes, once for each architecture, for every commit
I can remove the amd64 run if you think it's not necessary.
These are the 3 workflow runs from when I tested the change:
- amd64: https://github.com/antrea-io/antrea/actions/runs/11902027780 (7m46s)
- arm64: https://github.com/antrea-io/antrea/actions/runs/11902068843 (12m35s)
- arm/v7: https://github.com/antrea-io/antrea/actions/runs/11902071875 (13m28s)

I think this is ok because it only runs when main is updated, but let me know your thoughts.

Reply:
sounds good to me given it doesn't take a long time.